docs/experimental-features.md

```diff
@@ -25,6 +25,9 @@ Enables the ability to extend bicepparam files from other bicepparam files. For
 
 Allows Bicep to use an extensibility model to deploy non-ARM resources. Currently, we support Kubernetes extension ([Bicep Kubernetes extension](https://learn.microsoft.com/en-us/azure/azure-resource-manager/bicep/bicep-kubernetes-extension)) and Microsoft Graph extension ([Bicep templates for Microsoft Graph](https://aka.ms/graphbicep)).
 
+### `externalInputFunction`
+Enables the `externalInput` function to allow reading input that will be resolved later by external tooling. This is useful for example for reading environment variables, and inline-provided parameter values during deployment.
```

Comment:
I would avoid mentioning the specifics, just because we're not sure we're actually going to build support for things like env vars.

```suggestion
Enables the `externalInput` function to allow reading input that will be resolved later by external tooling. This is useful for tools that require the ability to inject values at deployment time.
```

Reply:
I think it's also worth adding a note to explain that this isn't going to work out-of-the-box for anyone. Here's what Tate added in his description for #16655: `(Note: This feature will not work until the backend service support has been deployed)`.